packages/cli/src/commands/generate/page/templates/page.tsx.template

```diff
@@ -1,7 +1,7 @@
 import { Link, routes } from '@redwoodjs/router'
 import { MetaTags } from '@redwoodjs/web'
 <% if (paramName !== '') { %>
-type ${pascalName}PageProps = {
+interface ${pascalName}PageProps {
```

Comment:
This is totally scope creep, so feel free to ignore, but personally I always just name the interface `Props`.

* Best practice is always to just have a single component in a file, so there's never any doubt what component the Props interface is used for.
* A shorter name makes it more likely that the entire function args list + type can all fit on a single line, greatly improving readability (imo).
* No mental overhead of trying to figure out what to name the prop type, or what its name is when using it. It's always just `Props`. (Less of an issue when using the generators, but at some point most people are probably going to write a component from scratch)

Reply:
Makes sense! I don't have a strong opinion. We're probably going to export these interfaces in the future so that the stories can use them. There's still no confusion there because there's only just one component, layout, page, etc. right?

Reply:
Yes. And, if there ever was risk of confusion when importing you can always use an alias import `Props as PostPageProps`